terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tfvars

```
@@ -0,0 +1,15 @@
+account_ids = {
+  analytical-platform-data-production       = "593291632749"
+  analytical-platform-management-production = "042130406152"
+}
+
+tags = {
+  business-unit          = "Platforms"
+  application            = "Data Platform"
+  component              = "Message Queue"
```

```suggestion
  component              = "Message Broker"
```
to match component name